### DISCUSSION TITLE
Power usage when running a Nvidia AGX Orin.

### DISCUSSION THREAD

**Discussion by valdivj** [iRobotEducation/create3_docs#165] [Ideas]
When I dock the I robot create 3 and the Nvidia Xavier NX is just idling doing  nothing there is plenty of power run the Xavier NX and recharge the I robot create 3 battery.
But when I hook up my Nvidia AGX Orin to the I robot create 3 and its docked with the AGX Orin idling there is just not enough juice to run the AGX Orin in idle and charge up the I robot create 3 battery at the same time.

Is there any thing I can do so I can use the Nvidia AGX Orin and the I robot create  3.

**Comment by shamlian** [comment]
I don't know the answer, but I would recommend posting on the NVIDIA customer support forums about putting this board into a low power mode. If [this article](https://www.hackster.io/news/the-cube-of-ai-power-a-hands-on-review-of-the-nvidia-jetson-agx-orin-developer-s-kit-0c48c240d54a) is to be believed, the device draws 19 W in its lowest power mode (marketed as 15 W) -- this is awfully high, but not impossibly so. The article also suggests a 5W mode is coming. How much current is coming out of the battery when the robot and device are idle on the dock (what does the [battery state topic](https://iroboteducation.github.io/create3_docs/examples/sensors-cli/#battery-state) report)? Also, how are you powering this device? I assume through VBAT? USB-C technically tops out at 15 W when running at 5 V (which Create 3 does). If you're right on the edge, you could try putting the Create 3 robot into [standby mode](https://iroboteducation.github.io/create3_docs/hw/face/#button-1).
